src/main/java/game/Answer.java

```diff
@@ -0,0 +1,22 @@
+package game;
+
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+public class Answer<T> {
+    private final Integer maxNumberOfElements;
+    private List<Element<T>> elements;
+
+    public Answer(Integer maxNumberOfElements){
+        this.maxNumberOfElements = maxNumberOfElements;
+        this.elements = new ArrayList<>();
+    }
+
+    public void addElement(Element<T> element){
+        if(3<elements.size()) throw new IllegalArgumentException();
```

Comment:
```suggestion
        if(3<elements.size()) throw new IllegalArgumentException();
```

3이라는 숫자는 어떠한 의미를 가지고 있을까요? 상수로 선언하여 변수명으로써 의미를 부여하면 보다 가독성이 좋을 것 같네요.!
이를 보통 매직넘버라 부릅니다 .!

[매직넘버란?](https://www.javatpoint.com/magic-number-in-java)

Reply:
3도 Setting 클래스에 별도로 분리하였습니다.